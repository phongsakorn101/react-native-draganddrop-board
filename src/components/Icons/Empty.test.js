import React from 'react'
import renderer from 'react-test-renderer'
import { Empty } from './index'

describe('Icon component', () => {
  describe('Renders correctly', () => {
    test('it renders Default Empty Icon', () => {
      const tree = renderer.create(
        <Empty
          color="#FFFFFF"
        />
      ).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
