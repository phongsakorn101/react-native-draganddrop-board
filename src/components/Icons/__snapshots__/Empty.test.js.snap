// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component Renders correctly it renders Default Empty Icon 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/assets/icons/emptyBox.png",
    }
  }
  style={
    Object {
      "height": 40,
      "tintColor": "#FFFFFF",
      "width": 40,
    }
  }
/>
`;
